# oneline-book-and-tutorial  
## book
[an introduction to human population genetics, variation and disease](https://web.stanford.edu/group/pritchardlab/HGbook.html)  

## singlecell
[hbctraining](https://github.com/hbctraining/scRNA-seq_online/tree/master/lessons) 
This repository has teaching materials for a hands-on Introduction to single-cell RNA-seq analysis workshop. This workshop will instruct participants on how to design a single-cell RNA-seq experiment, and how to efficiently manage and analyze the data starting from count matrices. This will be a hands-on workshop in which we will focus on using the Seurat package using R/RStudio. Working knowledge of R is required or completion of the Introduction to R workshop.  
[Single-cell best practices](https://www.sc-best-practices.org/introduction/prior_art.html)  
[Beginner’s Guide to Understanding Single-Cell ATAC-Seq](https://www.activemotif.com/blog-single-cell-atac-seq)  
[ArchR](https://www.archrproject.com/bookdown/getting-started-with-archr.html)  
[Signac](https://stuartlab.org/signac/)  
[SRT分析项目收集](https://docs.qq.com/doc/DVlJ6eEF1UEdDWlB2?groupUin=AJV9j8eDByKEseRLgXrvkA%253D%253D&ADUIN=441336361&ADSESSION=1686402498&ADTAG=CLIENT.QQ.5681_.0&ADPUBNO=27272&jumpuin=441336361)  
[kallisto bustools](https://www.kallistobus.tools/)  
is a workflow for pre-processing single-cell RNA-seq data. Pre-processing single-cell RNA-seq involves: (1) association of reads with their cells of origin, (2) collapsing of reads according to unique molecular identifiers (UMIs), and (3) generation of gene or feature counts from the reads to generate a cell x gene matrix.   
[Single-Cell Analysis Workflows with Bioconductor](https://bioconductor.org/books/3.17/OSCA.workflows/)  
This contains worked case studies of analyses of a variety of single-cell datasets, each proceeding from a SingleCellExperiment object. Exposition is generally minimal other than for dataset-specific justifications for parameter tweaks; refer to the other books in the OCSA collection for a detailed explanation of the theoretical basis of each step. It is intended for readers who already know the background and just want some code to copy and paste into their own analyses.  
[NCGR/INBRE](http://gateway.training.ncgr.org/single-cell-workshop/index.html#acknowledgments)  
Welcome to the NCGR/INBRE single cell RNA-seq analysis workshop. We will familiarize you with commonly useful R commands, the SingleCellExperiment and Seurat R packages, the Loupe Browser desktop application, and an example of a ShinyCell web application. These tools will give you basic experience in performing statistical analyses and data visualization for a variety of bioinformatic approaches to single cell sequencing.  
[Cellxgene VIP](https://interactivereport.github.io/cellxgene_VIP/tutorial/docs/index.html)  
Cellxgene VIP unleashes full power of interactive visualization and integrative analysis of scRNA-seq, spatial transcriptomics, and multiome data.  
[OUYANG-LAB](https://ouyanglab.com/singlecell/)  
aiming to explain the rationale behind different analysis step.    
[oshlacklab](http://oshlacklab.com/combes-organoid-paper/07_Combined_Clustering.html#introduction)  
In this document we are going to load the combined Organoids and Linstrom dataset and complete the rest of the Seurat analysis.  
[clustered-dotplot-for-single-cell-rnaseq](https://divingintogeneticsandgenomics.com/post/clustered-dotplot-for-single-cell-rnaseq/)  

## CUT&Tag
[yezheng](https://yezhengstat.github.io/)  
[CTCF](https://dozmorovlab.github.io/CTCF/)  
CTCF defines an AnnotationHub resource representing genomic coordinates of FIMO-predicted CTCF binding sites for human and mouse genomes, including the Telomere-to-Telomere and mm39 genome assemblies. It also includes experimentally defined CTCF-bound cis-regulatory elements from ENCODE SCREEN.  

## Other
[CS自学指南](https://csdiy.wiki/)
[luohao-brian](https://luohao-brian.gitbooks.io/gene_sequencing_book/content/di-1-8282-ce-xu-ji-zhu.html)
